src/platforms/zulip.md

```diff
@@ -1,13 +1,17 @@
 # Zulip
 
-[Rust's Zulip](https://rust-lang.zulipchat.com) is used by a number of teams, notably
-the compiler, language, and library teams, along with their working groups.
+[Rust's Zulip](https://rust-lang.zulipchat.com) is the primary communication channel of the Rust Project.
```

Comment:
I'm a little uncertain about the phrasing here. I think it depends on the perspective, but I wouldn't consider Zulip to be the primary communication channel. There are a few different considerations:

- One could say our primary communication channel to those outside of the project is the blog.
- I would suspect that there is actually more activity on GitHub, and our primary communication channel for tracking issues, active work, etc.

Maybe this means something closer to "the primary chat platform for Rust Project members to discuss project topics" or something along those lines?